events/api.py

```diff
@@ -1521,7 +1556,7 @@ def filter_queryset(self, queryset):
             if 'show_all' in self.request.query_params:
                 # Show all events for this organization,
                 # along with public events for others.
-                auth_filters |= Q(publisher=self.organization)
+                auth_filters |= get_publisher_query(self.organization)
```

Comment:
Here, `self.organization` is actually defined by `get_authenticated_data_source_and_publisher`, and there it is generated from the `User` model `get_default_organization`.

I believe that when we are doing this change, we already might want to change that method to not return replaced organizations? Even if this filter works because it checks the replacing organization, fixing the organization to be non-replaced will fix the `v1/user` endpoint as well, I believe.

Reply:
Make sense